admin/server/projects.go

```diff
@@ -165,7 +166,7 @@ func (s *Server) GetProject(ctx context.Context, req *adminv1.GetProjectRequest)
 		// The magic token's resource
 		condition.WriteString(fmt.Sprintf(" OR '{{.self.kind}}'=%s AND '{{lower .self.name}}'=%s", duckdbsql.EscapeStringValue(mdl.ResourceType), duckdbsql.EscapeStringValue(strings.ToLower(mdl.ResourceName))))
 		// If the magic token's resource is an Explore, we also need to include its underlying metrics view
-		if mdl.ResourceType == runtime.ResourceKindExplore {
+		if mdl.ResourceType == runtime.ResourceKindExplore || mdl.ResourceType == runtime.ResourceKindReport {
```

Comment:
For report tokens, could we avoid needing to grant access to the explore or underlying metrics view if we make the report download API only require access to the report itself?

Reply:
That works for the API however internally an export will be called on the underlying query which will require access to the mv